docs/development/plugin-triggers.md

```diff
@@ -856,19 +856,21 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 
 ### `nginx-dokku-template-source`
 
-- Description: Customize the source location of the `dokku.conf` nginx configuration file
+- Description: Return the path to a `sigil` template that should be used to generate the `dokku.conf` nginx configuration file.
 - Invoked by: `nginx-vhosts#install`
-- Arguments:
+- Arguments: None, however the `sigil` template can make use of the following variables: `$.DOKKU_ROOT $.NGINX_ROOT`
```

Comment:
That’s not the full list, might be worth linking to the nginx customization docs instead.

Reply:
Hmm, I think I might be missing something. 

- The ["Customizing the nginx Configuration"](http://dokku.viewdocs.io/dokku/configuration/nginx/#customizing-the-nginx-configuration) section of the docs only talks about customizing the application specific, not the global default.
- The call to `sigil` in the plugin only passes the two variables that I mentioned: 
https://github.com/dokku/dokku/blob/7c0eb20de8555a9f0065d0b70810b9710b276fdd/plugins/nginx-vhosts/install#L65-L71